libp2p/Session.cpp

```diff
@@ -319,10 +319,14 @@ void Session::send(bytes&& _msg)
 
 void Session::write()
 {
-	const bytes& bytes = m_writeQueue[0];
-	m_io->writeSingleFramePacket(&bytes, m_writeQueue[0]);
+	bytes *out;
```

Comment:
Pointer itself cannot be (`bytes* const`) but can point a const bytes (`bytes const*`).